v3/src/components/web-view/web-view-model.ts

```diff
@@ -37,15 +37,21 @@ export const WebViewModel = TileContentModel
   .views(self => ({
     get allowBringToFront() {
       return !self.preventBringToFront
+    },
+    get isGuide() {
+      return self.subType === "guide"
+    },
+    get isPlugin() {
+      return self.subType === "plugin"
     }
   }))
   .actions(self => ({
     setDataInteractiveController(controller?: iframePhone.IframePhoneRpcEndpoint) {
       self.dataInteractiveController = controller
     },
-    setIsPlugin(isPlugin: boolean) {
+    setSubType(subType: WebViewSubType) {
```

Comment:
Maybe you want to allow `undefined` here so the subtype can be reverted back to the default non plugin, non guide.